modules/lightning_telemetry/lightning_telemetry.services.yml

```diff
@@ -0,0 +1,4 @@
+services:
+  lightning_telemetry.telemetry:
```

Comment:
Since this is part of the Lightning suite, I think we can call this service `lightning.telemetry` to save ourselves some typing.